README.md

```diff
@@ -38,5 +40,7 @@ with [Meteor](https://www.meteor.com).
 [docker_image]: https://hub.docker.com/r/mquandalle/wekan/
 [heroku_button]: https://www.herokucdn.com/deploy/button.png
 [heroku_deploy]: https://heroku.com/deploy?template=https://github.com/wekan/wekan/tree/master
+[indiehosters_button]: https://indie.host/signup.png
```

Comment:
Yes there is, but I don't understand why Github doesn't want it :/ maybe the extension `.host` is not recognized, I'll ask Github support.